src/known-app-ids.js

```diff
@@ -0,0 +1,12 @@
+// These app IDs are generated from <name>.aragonpm.eth
+export default {
+  Agent: '0x9ac98dc5f995bf0211ed589ef022719d1487e5cb2bab505676f0d084c07cf89a',
```

Comment:
If we don't want to hardcode these, we could use `namehash('agent.aragonpm.eth')` :).

Reply:
Done! 3fe882c813ab0aa2d4bea5e69305b0fba6baa6e3